toto/toto-run.py

```diff
@@ -132,9 +132,9 @@ def main():
   product_list = []
 
   if args.materials:
-    material_list = args.materials.split(",")
+    material_list = args.materials
   if args.products:
-    product_list = args.products.split(",")
+    product_list = args.products
 
```

Comment:
Any reason not to add `args.materials` and `args.products` directly to the function call in [`in_toto_run`](https://github.com/in-toto/in-toto/pull/32/files#diff-8e16d631a0ef00bfa92de5d2ad6c1f65R139)?